### PR TITLE
Create user profile

### PR DESCRIPTION
**Pull request by distantfutures**
Implemented user profile and set up navigation to User Profile Activity using Intents. #6

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,18 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (riboney) [inline app/src/main/res/layout/activity_main.xml:2]: Whats the reason to nest the `constraintLayout` inside of `layout`?

**Review by riboney — APPROVED** [review]
Copy pasting from #6 :

- [X] Email
- [X] firstname
- [X] lastname
- [X] phone
- [X] skills
- [ ] Edit icon (optional)
- [X] Change password button
- [X] Forgot password button

Awesome work!

**Comment by riboney** [comment]
Pasting images of the android screens contributed by @distantfutures 

image

image

**Comment by oburnett127** [comment]
very good job!

**Comment by oburnett127** [comment]
PR closes issue #6

**Pull request merged**